starwars/src/components/info/InfoContainer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './info.css';
+
+const InfoContainer = (props) => {
+    return (
+        <div className="info-container">
+            <div className="name">
+                <h1>Name: <a href={props.starwarsChar.url}>{props.starwarsChar.name}</a></h1>
```

Comment:
The links here just link to the API call, so they aren't really useful for a user.